### PR TITLE
Fix #1687 - Improve formatting of Album class.

### LINKED ISSUE #1687: Improve formatting of Album class
**Expected Behaviour**

Properly format code of Album class.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mayank8318**
Fixed #1687 

Changes: Improve formatting and arrange variables properly.

Screenshots for the change:

### CODE REVIEW

**Comment by mayank8318** [comment]
@mohitmanuja @isuruAb Please review :)

File: app/src/main/java/org/fossasia/phimpme/gallery/data/Album.java [hunk 1]
```diff
@@ -33,631 +33,632 @@
...
+    }
+
+    private void scanFile(Context context, String[] path, MediaScannerConnection.OnScanCompletedListener onScanCompletedListener) {
+        MediaScannerConnection.scanFile(context, path, null, onScanCompletedListener);
+    }
+
+    private FilterMode getFilterMode() {
+        return settings != null ? settings.getFilterMode() : ALL;
+    }
 }
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/gallery/data/Album.java]: Add new line here.

**Comment by mayank8318** [comment]
@anantprsd5 Please review :)

**Comment by mariobehling** [comment]
Closing due to inactivity.

**Pull request closed without merging**